package-lock.json

```diff
@@ -44,46 +44,47 @@
       }
     },
     "node_modules/@babel/code-frame": {
-      "version": "7.24.7",
```

Comment:
I'd maybe discard the changes in this file as it looks like they might be causing the CI failures

Reply:
👍 I think any conflicts in the package-lock are now resolved.  Also had to bump the minimum node version to 16.x in sdk-generator and remove 14 support from the GH workflow.